mergekit/scripts/mixtral_moe.py

```diff
@@ -113,7 +113,7 @@ def tokenize_prompts(
     prompts: List[str], tokenizer: transformers.PreTrainedTokenizerBase
 ):
     return tokenizer(
-        [tokenizer.bos_token + p for p in prompts],
+        [str(tokenizer.bos_token) + p for p in prompts],
```

Comment:
It probably would be better to do `tokenizer.bos_token or ""` here - otherwise it'll be prepending the string "None" to your prompts if the BOS token isn't set.

Reply:
ok done